service/forward_service.cc

```diff
@@ -504,12 +504,12 @@ future<query::forward_result> forward_service::dispatch(query::forward_request r
     tracing::trace(tr_state, "Dispatching forward_request to {} endpoints", vnodes_per_addr.size());
 
     retrying_dispatcher dispatcher(*this, tr_state);
-    std::optional<query::forward_result> result;
```

Comment:
Commitlog says this commit extracts `requires_thread()` but that was apparently later split into a separate patch.

Reply:
Right, this was the first patch in the series, so please update the commit message